--- col.py	2012-07-16 11:29:49.000000000 -0700
+++ col2.py	2012-07-16 11:30:35.000000000 -0700
@@ -373,12 +373,12 @@
 
     baseClass = SOCol
 
-    def __init__(self, name=None, **kw):
+    def __init__(self, name=None, extra_vars={}, **kw):
         super(Col, self).__init__()
         self.__dict__['_name'] = name
         self.__dict__['_kw'] = kw
         self.__dict__['creationOrder'] = creationOrder.next()
-        self.__dict__['_extra_vars'] = {}
+        self.__dict__['_extra_vars'] = extra_vars
 
     def _set_name(self, value):
         assert self._name is None or self._name == value, (
